src/Common/src/Interop/Windows/Kernel32/Interop.QueryPerformanceCounter.cs

```diff
@@ -2,14 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Runtime.InteropServices;
 
 internal partial class Interop
 {
     internal partial class Kernel32
     {
-        [DllImport(Libraries.Kernel32)]
-        internal static extern bool QueryPerformanceCounter(out long value);
+        // The actual native signature is:
+        //      BOOL WINAPI QueryPerformanceCounter(
+        //          _Out_ LARGE_INTEGER* lpPerformanceCount
+        //      );
+        //
+        // We return an int (rather than a bool) to avoid the marshalling overhead.
```

Comment:
We have `BOOL` in https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/Interop/Windows/Interop.BOOL.cs for cases like these and not needing to explain the mismatch in every place.

Reply:
The `BOOL` doesn't exist on Unix, however, and you have the same problem of needing to explain that the `int` is actually a 4-byte `bool`

Reply:
Actually, looks like I could just include it and have it used for the UNIX signatures as well.

Reply:
Fixed.